zschema/tests.py

```diff
@@ -30,6 +31,26 @@ def test_invalid(self):
             except DataValidationException:
                 continue
 
+    def test_to_dict(self):
```

Comment:
Would be good to have these for compound types too; I actually found that problem because I added calls to `docs_es` / `docs_bq` in addition to `validate` in my zgrab tests, so the compounds will be executed there, but it's not systematic...it's just whatever happens to be used in those schemas.